app/controllers/concerns/form_attachment_create.rb

```diff
@@ -18,18 +19,33 @@ def serializer_klass
   end
 
   def validate_file_upload_class!
-    # is it either ActionDispatch::Http::UploadedFile or Rack::Test::UploadedFile
-    unless filtered_params[:file_data].class.name.include? 'UploadedFile'
-      raise Common::Exceptions::InvalidFieldValue.new('file_data', filtered_params[:file_data].class.name)
+    begin
```

Comment:
Looks like you don't need `begin` here as it is redundant. Once these are cleaned up it looks good to go 👍